packages/angular_devkit/build_angular/src/utils/routes-extractor/extractor.ts

```diff
@@ -53,13 +55,21 @@ async function* getRoutesFromRouterConfig(
 
     yield { route: currentRoutePath, success: true, redirect: false };
 
-    if (children?.length || loadChildren) {
-      yield* getRoutesFromRouterConfig(
-        children ?? (await routerConfigLoader.loadChildren(injector, route).toPromise()).routes,
-        routerConfigLoader,
-        injector,
-        currentRoutePath,
-      );
+    if (children?.length) {
+      yield* getRoutesFromRouterConfig(children, compiler, parentInjector, currentRoutePath);
+    }
+
+    if (loadChildren) {
+      const loadedChildRoutes = await loadChildrenHelper(
+        route,
+        compiler,
+        parentInjector,
+      ).toPromise();
+
+      if (loadedChildRoutes) {
+        const { routes: childRoutes, injector = parentInjector } = loadedChildRoutes;
```

Comment:
the returned injector might be `undefined` here

Reply:
Yeah, I am
Aware of that, and that’s why it defaults to the `parentInjector`

Reply:
Oh, I just was totally unfamiliar with the syntax 😅